src/rpc/blockchain.cpp

```diff
@@ -1185,20 +1182,21 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
 
     LOCK(cs_main);
 
+    const CBlockIndex* tip = chainActive.Tip();
```

Comment:
No need to change these unless you're gonna actually kill the cs_main held for the whole time, I'd think, no (and little reason to, its not like its being held for an extended period)?

Reply:
This moves up tip variable declared below.